lib/jekyll/configuration.rb

```diff
@@ -177,6 +177,7 @@ def read_config_files(files)
 
       begin
         files.each do |config_file|
+          next if config_file.nil? or config_file.empty?
```

Comment:
If you send `{ "config" => "" }`, we try to load the file. That's not such a great thing. What's a better way which preserves some of the defensiveness?